src/capella/withdrawal.rs

```diff
@@ -0,0 +1,11 @@
+use crate::primitives::{ExecutionAddress, Gwei, ValidatorIndex, WithdrawalIndex};
+use ssz_rs::prelude::*;
+
+#[derive(Default, Debug, Clone, SimpleSerialize, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct Withdrawal {
+    pub index: WithdrawalIndex,
+    pub validator_index: ValidatorIndex,
+    pub address: ExecutionAddress,
+    pub amount: Gwei,
```

Comment:
going to want `serde` annotations:
```suggestion
    #[serde(with = "crate::serde::as_string")]
    pub index: WithdrawalIndex,
    #[serde(with = "crate::serde::as_string")]
    pub validator_index: ValidatorIndex,
    pub address: ExecutionAddress,
    #[serde(with = "crate::serde::as_string")]
    pub amount: Gwei,
```